service/src/test/java/tds/assessment/repositories/impl/ItemQueryRepositoryImplIntegrationTests.java

```diff
@@ -187,23 +181,13 @@ public void shouldRetrieveActiveItemPropsForNonSegAssesssment() {
     }
 
     @Test
-    public void shouldRetrieveSingleItemNonSegmentedTest() {
+    public void shouldRetrieveSingleItemNonSegmentedAdaptiveTest() {
         List<Item> items = repository.findItemsForAssessment("(SBAC_PT)IRP-Perf-ELA-11-Summer-2015-2016");
-        assertThat(items).hasSize(2);
-
-        Item item1 = null;
-        Item item2 = null;
+        assertThat(items).hasSize(1);
 
-        for (Item item : items) {
```

Comment:
I removed the second item from the assessment test data and didn't feel like adding it back in since I didn't feel it added much value.